### PR TITLE
Hide Active Theme's Delete Button

### PR DESCRIPTION
**Pull request by DanHarrin**
This PR hides the active theme's delete button from its "Manage" dropdown menu.

At the moment, the button is still clickable and warns you that the action "can't be undone". When the user proceeds, it informs you that you can't delete an active theme! This is not very intuitive in my opinion.

### CODE REVIEW

**Comment by w20k** [comment]
Done, tested -  ✅ 

Before:
Screenshot 2019-08-07 at 16 30 54
After:
Screenshot 2019-08-07 at 16 30 45

File: modules/cms/controllers/themes/_theme_list_item.htm [hunk 1]
```diff
@@ -100,19 +100,21 @@ <h3><?= e($theme->getConfigValue('name', $theme->getDirName())) ?></h3>
...
-                        tabindex="-1"
-                        data-request="onDelete"
-                        data-request-confirm="<?= e(trans('cms::lang.theme.delete_confirm')) ?>"
-                        data-request-data="theme: '<?= e($theme->getDirName()) ?>'"
-                        href="javascript:;"
-                        class="oc-icon-trash">
-                        <?= e(trans('cms::lang.theme.delete_button')) ?>
-                    </a>
-                </li>
+                <?php if (! $theme->isActiveTheme()): ?>
```
Comment (LukeTowers) [inline modules/cms/controllers/themes/_theme_list_item.htm]:
```suggestion
                <?php if (!$theme->isActiveTheme()): ?>
```

**Comment by LukeTowers** [comment]
@DanHarrin one minor change and then I should be able to merge this

**Comment by DanHarrin** [comment]
@LukeTowers all done :+1:

**Pull request merged**